From 002160f6d269b48cb80a9eea6fbb5c3c2c4511ac Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Sun, 16 Jul 2023 21:19:15 +0300
Subject: [PATCH] [SW-152708] gaudi3: make: fix h9 includes

The SPECS_H9_DIR variable has moved to point to H9 specs base directory.
Therefore add the relevant (registers directory) suffix.
Also add the base path to include directories to match with older
devices specs.

Change-Id: Icc594642c10d76f7b69c55ab6a63a4abcb764647
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/arch/gaudi3/common/Makefile b/arch/gaudi3/common/Makefile
index f98ae03..159d90a 100644
--- a/arch/gaudi3/common/Makefile
+++ b/arch/gaudi3/common/Makefile
@@ -1,4 +1,5 @@
-CFLAGS += -I $(SPECS_H9_DIR)
+CFLAGS += -I$(SPECS_H9_DIR)
+CFLAGS += -I$(SPECS_H9_DIR)/gaudi3/asic_reg
 
 PPBOOT_SRCS_C += $(shell ls $(HL_DEV_ARCH_COMMON)/common.c)
 PPBOOT_SRCS_C += $(shell ls $(HL_DEV_ARCH_COMMON)/$(HL_TARGET).c)
